versions.yaml

```diff
@@ -109,6 +109,10 @@ assets:
     description: "Linux kernel optimised for virtual machines"
     url: "https://cdn.kernel.org/pub/linux/kernel/v4.x/"
     version: "v4.14.67"
+    dev:
+      description: "dev-kernel for virtual machines"
+      url: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/snapshot/"
```

Comment:
I copied and pasted this url in the browser, I got next message:

`No snapshot name specified`

is this expected?

Reply:
my fault. `/snapshot` should be removed.
But since the `rc` version is unacceptable, i need to drop this solution.